ossdbtoolsservice/language/language_service.py

```diff
@@ -179,19 +175,11 @@ def handle_flavor_change(self,
         """
         if params is not None and params.uri is not None:
             if params.language.lower() == 'sql':
-                # provider.flavor can be PGSQL, MySQL, MSSQL
-                if params.flavor.lower() == utils.constants.PG_PROVIDER_NAME.lower():
-                    self._provider_valid_uri[utils.constants.PG_PROVIDER_NAME].add(params.uri)
+                # provider.flavor can be PGSQL, MySQL
```

Comment:
you mean params.flavor? also it can be other things too but its just that we support MySQL and PG.